.github/workflows/ci_test.yml

```diff
@@ -120,6 +115,7 @@ jobs:
           pip install -r requirements.txt
           pip install -e .
           python docker/build.py tf2
+          echo "armory version: $(armory --version)"
```

Comment:
Remove, unless you think it makes sense to add to the other docker tests.

Reply:
done.